src/screens/WalletScreen/index.tsx

```diff
@@ -45,6 +46,7 @@ const WalletScreen: React.FC<any> = ({ navigation, route }) => {
   const initializeWallet = useInitializeWallet();
   const { network: currentNetwork, accountAddress, appIcon, nativeCurrency } = useAccountSettings();
   usePositions({ address: accountAddress, currency: nativeCurrency });
+  useClaimables({ address: accountAddress, currency: nativeCurrency });
```

Comment:
should we add `{ notifyOnChangeProps: [] }` to these queries to avoid rerendering this whole screen when it refetches or is it supposed to?

Reply:
i dont think it triggers a rerender bc the return value of `useClaimables` is not stored